src/pip/_internal/index/collector.py

```diff
@@ -450,7 +451,7 @@ def _get_html_page(link, session=None):
     try:
         resp = _get_html_response(url, session=session)
     except _NotHTTP:
-        logger.debug(
+        logger.warning(
             'Skipping page %s because it looks like an archive, and cannot '
             'be checked by HEAD.', link,
```

Comment:
HEAD is a very ambiguous word here IMO. It'll help to clarify which HEAD we're talking about, if we're making this a more visible message. :)

```suggestion
            'be checked by a HTTP HEAD request.', link,
```

Reply:
That's a good idea. Done in https://github.com/pypa/pip/pull/8465/commits/6c899ed0c88dba4e1576bfb8515737fc07418951 . This should be ready to merge once green :)